### PR TITLE
feat(readImageDICOMFileSeries): Use GDCM instead of DCMTK

### PR DESCRIPTION
**Pull request by thewtex**
More transfer syntax support.

### CODE REVIEW

File: test/Browser/DICOMSeriesTest.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ test('Test reading DICOM file series', t => {
     })
     .then(function ({ image, webWorkerPool }) {
       t.is(image.imageType.dimension, 3, 'dimension')
-      t.is(image.imageType.componentType, IntTypes.UInt8, 'componentType')
+      t.is(image.imageType.componentType, IntTypes.Int16, 'componentType')
```
Comment (agirault) [inline test/Browser/DICOMSeriesTest.js:27]: Why would those change? The pixel type should be defined by [PixelRepresentation](https://dicom.innolitics.com/ciods/ct-image/image-pixel/00280103) and [BitsAllocated](https://dicom.innolitics.com/ciods/ct-image/image-pixel/00280100).

File: test/DICOMSeriesTest.js [hunk 2]
```diff
@@ -70,7 +70,8 @@ test('Test reading a DICOM file with the synchronous API', t => {
 test('Test reading a DICOM file with the synchronous API, assume sorted', t => {
   const image = readImageLocalDICOMFileSeriesSync(testFiles.sort(), true)
   t.is(image.imageType.dimension, 3, 'dimension')
-  t.is(image.imageType.componentType, IntTypes.UInt8, 'componentType')
+  t.is(image.imageType.componentType, IntTypes.Int16, 'componentType')
+  t.is(image.imageType.pixelType, PixelTypes.Scalar, 'pixelType')
```
Comment (agirault) [inline test/DICOMSeriesTest.js:74]: duplicated line `t.is(image.imageType.pixelType, PixelTypes.Scalar, 'pixelType')`

File: test/Browser/DICOMSeriesTest.js (hunk 1 above)
Comment (thewtex) [reply test/Browser/DICOMSeriesTest.js:27]: Likely related to how rescale slope / intercept is applied.

File: test/DICOMSeriesTest.js (hunk 2 above)
Comment (thewtex) [reply test/DICOMSeriesTest.js:74]: Good catch and thanks for the review! Removed here: https://github.com/InsightSoftwareConsortium/itk-js/pull/322

**Pull request merged**